### PR TITLE
Fix/refactoring course controller

### PR DESCRIPTION
**Pull request by m24murray**
- refactoring course controllers out of `homeController.ts` 🔀
- fixing broken `getCourse` (missing promise await) 🤔
- updating page titles and nav bars with nunjucks components 🥋
- various`en.json` translations 📖
- removing redundant `loadCourse` method and params functionality 🥊

### CODE REVIEW

File: src/controllers/courseController.ts [hunk 1]
```diff
@@ -23,15 +23,17 @@ export class CourseController {
...
-			const course = req.course
+		logger.debug('Course Overview page')
+		const self = this
 
-			response.render(`page/course`, {
-				course,
-			})
+		return async (request: Request, response: Response) => {
+			const courseId: string = request.params.courseId
+			const course = await self.learningCatalogue.getCourse(courseId)
```
Comment (elxsy) [inline src/controllers/courseController.ts]: INFO - why is loadCourse public?
INFO - Loading a course or 404 logic is repeated (with template variation)

File: src/locale/en.json [hunk 2]
```diff
@@ -1,5 +1,10 @@
 {
-	"home.page.title": "Content management - Civil Service Learning",
+	"content_management_title": "Content management",
+	"add_course_title_title": "Add course title",
+	"add_course_details_title": "Add course title",
```
Comment (elxsy) [inline src/locale/en.json]: INFO - same translation for two different key values?

File: views/component/Page.njk [hunk 3]
```diff
@@ -55,32 +55,32 @@
...
-      {
-        href: "#5",
-        text: "Admin"
-      },
-      {
-        href: "#6",
-        text: "Sign out"
-      }
-    ]
+               {
```
Comment (elxsy) [inline views/component/Page.njk]: STYLING - Either 2 spaces or 1 tab for everyone. Keep it consistent please!

Comment (m24murray) [reply views/component/Page.njk]: Cool will keep an eye on this going forward. I've reformatted this to look a little cleaner

File: src/locale/en.json (hunk 2 above)
Comment (m24murray) [reply src/locale/en.json]: Fixed!

**Pull request merged**